vagga.yaml

```diff
@@ -23,7 +23,7 @@ containers:
         - libpq
       - !Py3Requirements requirements.txt
     environ:
-      DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/mainDB
+      DATABASE_URL: sqlite://vagga:vagga@127.0.0.1:5433/mainDB
```

Comment:
That was my previous fault.
This container can't run postgres DB. SQLite is ok here